Custom tx timeout can be set to any value (#2282)
neo4j/docs-operations: modules/ROOT/pages/database-internals/transaction-management.adoc

====

Configuring transaction timeout does not affect transactions executed with custom timeouts (e.g., via the Java API or Neo4j Drivers), as the custom timeout overrides the value set for `db.transaction.timeout`.

> You can set the transaction timeout to any value, even larger than configured by `db.transaction.timeout`.

Reviewer: ```suggestion
Custom timeouts can be set to any value, even larger than configured by `db.transaction.timeout`.
```
I find "the value" ambiguous after talking about two different timeout values before.
